src/utils/index.js

```diff
@@ -0,0 +1,18 @@
+// TODO: edit to use real back end token instead of test key (TestLogin)
+const TOKEN_KEY = 'jwt';
```

Comment:
This is bundled in with [PR#2](https://github.com/c9y7ip/ezpark-admin/pull/2/files)